flink-filesystems/flink-s3-fs-presto/pom.xml

```diff
@@ -45,6 +45,14 @@ under the License.
 			<scope>provided</scope>
 		</dependency>
 
+		<!-- S3 base (bundled) -->
```

Comment:
This was moved up to make it's transitive dependencies appear in the right place in the dependency tree, wasn't it?

Reply:
correct :+1: